packages/eslint-tool/README.md

```diff
@@ -0,0 +1,56 @@
+## @dotcom-reliability-kit/eslint-tool
+
+A prototype linting tool, specifically focussed on enhancing code quality and proactively catching errors/bugs before they make it into production. This is not a replacement for the [@financial-times/eslint-config-next](https://github.com/Financial-Times/eslint-config-next), but it can be installed alongside it or as an extension in it's config. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+- [Usage](#usage)
+  - [Static Code Analysis](#static-code-analysis)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Usage
+
+---
+
+Install @dotcom-reliability-kit/eslint-tool as a dependency:
+
+```
+npm install --save @dotcom-reliability-kit/eslint-tool
+```
```

Comment:
**suggestion:** we should make sure we say to install this as a development dependency, we don't want it to be installed in production. So:

> Install @dotcom-reliability-kit/eslint-tool as a development dependency:
>
> ```
> npm install --save-dev @dotcom-reliability-kit/eslint-tool
> ```